packages/lesswrong/lib/collections/doppelCommentVotes/collection.ts

```diff
@@ -0,0 +1,18 @@
+import { addUniversalFields, getDefaultMutations, getDefaultResolvers } from "@/lib/collectionUtils";
+import { createCollection } from "@/lib/vulcan-lib";
+import schema from "./schema";
+
+const DoppelCommentVotes: DoppelCommentVotesCollection = createCollection({
+  collectionName: "DoppelCommentVotes",
+  typeName: "DoppelCommentVote",
+  schema,
+  logChanges: true,
+  resolvers: getDefaultResolvers('DoppelCommentVotes'),
+  mutations: getDefaultMutations('DoppelCommentVotes'),
```

Comment:
The mutations should have a set of mutation options controll who can create/edit/remove doppelCommentVotes via the generated default mutations (probably only logged in users can create, userIsAdmin or userOwns for edit, nobody (false) for remove)

Reply:
DoppelCommentVotes will also want a `checkAccess` function for enforcing read access on the collection level (rather than the field level)